### DISCUSSION TITLE
Feed filters do not work

### DISCUSSION THREAD

**Discussion by DealsBeam** [community/community#121304] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I want to only show the releases on my feed but selecting it doesn't show anything...

![image](https://github.com/community/community/assets/17716841/73bd65a3-c5c5-40f0-a5dd-d8b27b70ac67)
![image](https://github.com/community/community/assets/17716841/8d625fcf-8545-4422-9906-896cea43f42b)

**Comment by LiteBrite82** [comment]
This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
